apps/opensearch/setup_models.py

```diff
@@ -534,11 +534,19 @@ def create_pipelines(openai_id):
                     "combination": {"technique": "arithmetic_mean", "parameters": {"weights": [0.111, 0.889]}},
                 }
             }
-        ]
+        ],
+        "response_processors": [
+            {
+                "remote_processor": {
+                    "endpoint": "rps:2796/RemoteProcessorService/ProcessResponse",
```

Comment:
you can also specify this with a https:// prefix but default is ssl. (http://blah also works if you run rps in insecure mode - ie no certs)